techs/zetapack/factions/persian/units/snake_basket/snake_basket.xml

```diff
@@ -85,7 +85,7 @@
 			</sound>
 			<attack-strength value="120"/>
 			<attack-var value="50"/>
-			<attack-range value="12"/>
+			<attack-range value="16"/>
```

Comment:
I feel there should be an agreed-upon max attack range, that can't be exceeded by any units in ZetaPack. I saw the specialists defense tower was brought down from 20 to 15 recently. Are there any other units with an attack range higher than 15? Should 15 be the max? Does anyone else feel that we should agree upon a max attack range, and if so, what should it be?

Reply:
Yes, and it should probably be lower than 15, leaving 15 for very very special cases. I will lower the power_golem range to 14 or 13 on the next patch. I was so tired of magic being always the weakest faction by far that maybe I went too far with the range there. Very few things should outrange defensive buildings like the tech defense tower IMO.

Reply:
16 was a mistake, I meant to increase it to 15. I increased it to 15[16] so it can attack the catapults.

Reply:
ok